### PR TITLE
submitting homework for CSS exercise.

### PR DESCRIPTION
**Pull request by Rangermyke**

### CODE REVIEW

File: michael-le.html [hunk 1]
```diff
@@ -3,19 +3,20 @@
...
     <title>CSS Timeline</title>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="css/michael-le.css">
   </head>
   <body>
     <h1>Brief History of CSS</h1>
     <ul>
-      <li>1994: The first work on style sheets started at CERN.</li>
-      <li>December 1996: CSS was proposed as W3C standard:
+      <li id="green">1994: The first work on style sheets started at CERN.</li>
```
Comment (zac-heisey) [inline michael-le.html]: Don't forget to make your date in this line of code **bold** and _italicized_

File: michael-le.html [hunk 2]
```diff
@@ -3,19 +3,20 @@
...
+      <li ><span class="dates">December 1996</span>: CSS was proposed as W3C standard:
```
Comment (zac-heisey) [inline michael-le.html]: Careful with your spacing between `<>` tags.

File: css/michael-le.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  color: red;
+}
+#green {
+  color: green;
+}
+.dates {
+  font-weight: bold;
+  font-style: italic;
+}
+ul > li {
```
Comment (zac-heisey) [inline css/michael-le.css]: Check out the updated task for using a `child selector` in this assignment, and update this CSS rule.

**Review by zac-heisey — CHANGES_REQUESTED** [review]
Hey Michael, take a look at the requested changes and let me know if you have any questions. Thanks!

**Comment by Rangermyke** [comment]
Hey Zac, I have commited and pushed all the requested changes.

File: css/michael-le.css [hunk 4]
```diff
@@ -8,9 +8,12 @@ h1 {
...
-ul > li {
-  color: pink;
+li > blockquote {
+  text-decoration: underline;
 }
 body blockquote {
   color: grey;
 }
+body {
+  font-family: "roboto"
```
Comment (zac-heisey) [inline css/michael-le.css]: missing an ending `;` after this CSS declaration.

**Review by zac-heisey — CHANGES_REQUESTED** [review]
All of your fixes are looking good! Just a small tweak to be made.

**Review by zac-heisey — APPROVED** [review]

**Pull request merged**